vault/request_handling.go

```diff
@@ -582,13 +582,16 @@ func (c *Core) handleCancelableRequest(ctx context.Context, req *logical.Request
 			if token == nil {
 				return logical.ErrorResponse("invalid token"), logical.ErrPermissionDenied
 			}
-			// We don't care if the token is an server side consistent token or not. Either way, we're going
+			// We don't care if the token is a server side consistent token or not. Either way, we're going
 			// to be returning it for these paths instead of the short token stored in vault.
 			requestBodyToken = token.(string)
 			if IsSSCToken(token.(string)) {
 				token, err = c.CheckSSCToken(ctx, token.(string), c.isLoginRequest(ctx, req), c.perfStandby)
+
+				// If we receive an error from CheckSSCToken, we can assume the token is bad somehow, and the client
+				// should receive a 403 bad token error like they do for all other invalid tokens.
 				if err != nil {
-					return nil, fmt.Errorf("server side consistent token check failed: %w", err)
+					return logical.ErrorResponse(fmt.Sprintf("bad token: %s", err.Error())), logical.ErrPermissionDenied
```

Comment:
Not sure it's a good idea to put the actual error message into the response, helpful though it would be.

Reply:
I'm not sure we should log it either.  Since this can be done without authentication, it could allow a bad actor to fill up log devices.

Reply:
@ncabatoff we already have logs being generated as part of the SSCT checks if those checks fail. I'm not too sure what to do here, since on one hand we don't want to leak data about the token, but on the other hand debugging failing requests will be much more difficult if we don't have the information in the logs that point to exactly what is failing (lagging WALs, wrong key, etc.). What do you suggest we do?

Reply:
Ok, I'll remove the logging.